### DISCUSSION TITLE
ABS doesn't find any author info and even after adding ASIN for book, no info found?

### DISCUSSION THREAD

**Discussion by ReaderGuy42** [advplyr/audiobookshelf#2019] [Q&A]
About half of my books were seemingly successfully recognized and got cover images. The others didn't and even after adding ASINs for those, nothing happens?

And for authors absolutely none were recognized. 

And when I try manually Match-ing a book via that menu it doesn’t find anything even though I can go to audible and find the book.

What am I doing wrong?

Thanks!

**Comment by advplyr** [comment]
You may be expecting something to happen that isn't intended to happen automatically. You can also ask in the discord/matrix server if you are still unsure.
